spring-integration-scripting/src/main/java/org/springframework/integration/scripting/jsr223/DefaultScriptExecutor.java

```diff
@@ -30,7 +30,7 @@
 	 * Create a DefaultScriptExceutor for the specified language name (JSR233
 	 * alias).
 	 */
-	public DefaultScriptExecutor(String language) {
```

Comment:
These `ScriptExecutor` implementors are `public` candidates, too. 
For better Java Config coverage. 
See `ScriptExecutingMessageProcessor`.